zokrates_core/src/proof_system/zexe/mod.rs

```diff
@@ -0,0 +1,306 @@
+pub mod gm17;
+
+use crate::ir::{CanonicalLinComb, Prog, Statement, Witness};
+use zexe_gm17::Proof;
+use zexe_gm17::{
+    create_random_proof, generate_random_parameters, prepare_verifying_key, verify_proof,
+    Parameters,
+};
+
+use crate::flat_absy::FlatVariable;
+use algebra_core::PairingEngine;
+use r1cs_core::{
+    ConstraintSynthesizer, ConstraintSystem, ConstraintSystemRef, LinearCombination,
+    SynthesisError, Variable,
+};
+use std::collections::BTreeMap;
+use zokrates_field::{Field, ZexeFieldExtensions};
+
+pub use self::parse::*;
+
+use rand_0_7::SeedableRng;
```

Comment:
cf feature guard on rand_0_7

Reply:
I don't this is necessary here as the whole zexe module is under a feature guard? I might be wrong here

Reply:
Correct